students/csdotson/lesson04/file_lab.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python3
+import pathlib
+
+# Print full path for all files in cwd
+pth = pathlib.Path.cwd()
+for index, f in enumerate(pth.iterdir()):
+    print(f'File {index + 1}: ', f)
+
+# Copy files from a source to a destination
+source_file = 'file_to_move.txt'
+destination = pathlib.Path.home()
+outfile = open(destination / 'output.txt', 'w')
+
+for line in open(source_file):
+    outfile.write(line)
+outfile.close()
```

Comment:
couple notes here:

 * prefer to use `os.path.join` to create file path ([read here](https://stackoverflow.com/questions/13944387/why-use-os-path-join-over-string-concatenation))
* use `with` when opening files, here you're not closing source_file and if write crashes, outfile will be not be closed as well.